helm/hpcc/templates/_helpers.tpl

```diff
@@ -713,6 +713,9 @@ Add extra args for a component
  {{- $debugPlane := .me.debugPlane | default (include "hpcc.getFirstPlaneForCategory"  (dict "root" .root "category" "debug")) -}}
  {{- include "hpcc.checkPlaneExists" (dict "root" .root "planeName" $debugPlane) -}}
  {{- $prefix := include "hpcc.getPlanePrefix" (dict "root" .root "planeName" $debugPlane) -}}
+  {{- if or (and (hasKey .me "expert") .me.expert.alwaysPostMortem) (and (hasKey .root.Values.global "expert") .root.Values.global.expert.alwaysPostMortem) -}}
```

Comment:
This isn't short circuited on old versions of helm. 
MeExpert := .me.expert | dict
?

Reply:
hm, yes good point, will change.